stitching_base/qr-lipsync.sh

```diff
@@ -8,5 +8,6 @@ cd qr-lipsync
 # default python 3
 python3 setup.py install
 
-# python 3.6
+# python 3.6 (adding setuptools for arm64's python3.6)
+python3.6 -m pip install setuptools
```

Comment:
I would suggest like this.

```suggestion
if which python3.6
then
    python3.6 -m pip install setuptools
    python3.6 setup.py install
fi
```